### PR TITLE
collecting the ducks and listing them with their images (need to add the .pyc files to .ignore file)

### PR DESCRIPTION
**Pull request by Othmanovic**
implemented the feature where a duck found through the spin is added to the user's profile and displayed along with its image

1. Updated the spin_duck view
Modified the spin_duck function to assign the selected duck to the currently logged-in user by creating a new Auction

2. Updated the profile view
Ensured the profile view fetches ducks from the auctions created for the logged-in user.

3. Updated profile.html
Ensured the template correctly displays the duck's name, rarity, and image.

### CODE REVIEW

**Review by ermias19 — APPROVED** [review]

File: api/__pycache__/__init__.cpython-312.pyc
Comment (PavlovaAlexandra) [inline api/__pycache__/__init__.cpython-312.pyc:1]: Do we need to push this folder? Maybe we can delete it from this pr?

File: api/migrations/__pycache__/0001_initial.cpython-312.pyc
Comment (PavlovaAlexandra) [inline api/migrations/__pycache__/0001_initial.cpython-312.pyc:1]: Same for this folder

File: gacha_game/__pycache__/__init__.cpython-312.pyc
Comment (PavlovaAlexandra) [inline gacha_game/__pycache__/__init__.cpython-312.pyc:1]: And there

File: api/__pycache__/__init__.cpython-312.pyc
Comment (ermias19) [reply api/__pycache__/__init__.cpython-312.pyc:1]: Yes you are correct @PavlovaAlexandra, it should not belong here. we really have to add it in .gitignore

**Comment by ermias19** [comment]
@Othmanovic would you resolve the merge conflict of these ** api/views.py
db.sqlite3
gacha_game/urls.py
templates/profile.html
templates/spin_result.html **

**Comment by PavlovaAlexandra** [comment]
I removed _pycache_ and fixed conflicts. Closing this pr

**Pull request merged**